benchkit/utils/dir.py

```diff
@@ -249,4 +250,4 @@ def caller_file_abs_path(
 def caller_dir() -> pathlib.Path:
     caller_filepath = caller_file_abs_path()
     caller_parent = caller_filepath.parent.resolve()
-    return caller_parent
+    return caller_parent
```

Comment:
```suggestion
    return caller_parent

```

All text files should end with `\n`.

Reply:
something is still odd here.. i'm going to let that pass and add a checker